packages/core/src/chains/sol/sdkSol/OffchainMessage.ts

```
@@ -130,6 +138,308 @@ export class OffchainMessage {
+  static createOffChainMessage({
+    message,
+    applicationDomain,
+    signerPublicKeys = [],
+    format = 0,
+    isLegacy = false,
+  }: ICreateOffChainMessageOptions) {
+    if (message.length === 0) {
+      throw new Error('Message cannot be empty');
+    }
+
+    const messageBytes = this.createMessageBytes(message, format);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)